tests/test_vault.py

```diff
@@ -0,0 +1,185 @@
+from ocw.lib.vault import Vault, AzureCredential, EC2Credential, GCECredential
+from datetime import datetime
+from datetime import timedelta
+import pytest
+import requests
+import webui
+import json
+import string
+import random
+import base64
+
+
+# Global test data
+namespace = 'test-qa'
+host = 'http://foo.bar'
+leases = list()
+
+
+def random_str(stringLength):
```

Comment:
for me it is reinventing the wheel . there are tons of classes providing such functionality . I personaly prefer  something like this https://docs.python.org/3/library/uuid.html , but in case UUID does not fit your model I am sure we can find something what generating just letter string

Reply:
nice, I will use it.